src/Installer/Form/ModuleConfigureForm.php

```diff
@@ -65,6 +65,8 @@ protected function getEditableConfigNames() {
    */
   public function buildForm(array $form, FormStateInterface $form_state) {
 
+    $this->messenger()->deleteAll();
```

Comment:
Let's add a comment as to why we're doing this and also link to an issue to fix this.